openwisp_monitoring/monitoring/tasks.py

```diff
@@ -0,0 +1,12 @@
+from celery import shared_task
+
+from ..db import timeseries_db
+from ..db.exceptions import TimeseriesWriteException
+
+
+@shared_task(bind=True, autoretry_for=(TimeseriesWriteException,), retry_backoff=True)
```

Comment:
looks good here, we need to be able to specify more options here and be able to configure them by settings:
https://github.com/openwisp/openwisp-firmware-upgrader/#openwisp_firmware_upgrader_retry_options